### ISSUE TITLE
Data transformation

### ISSUE THREAD

**Issue by muhammedcanpirincci-sudo** [bigmb/Unet-Segmentation-Pytorch-Nest-of-Unets#53]
Hello. Thanks for sharing this beautiful work.
I am using this dataset: https://www.kaggle.com/c/carvana-image-masking-challenge
When I delete cropping on transformation everything works great, but when I don't delete, predictions are being incredibly awful.
So in each iteration, I am changing my Dataloader randomly (in epoch for-loop). The batch will either will consist of cropped or not chopped pictures. Is my approach correct? What am I doing wrong do you think my images that cropped are giving me terrible results?

**Comment by bigmb** [comment]
Thanks. 
By crop do you mean center crop? 
Also did you try visualize the images , which are cropped and uncrowded before sending them to training? If yes, can you share me the images?

**Comment by muhammedcanpirincci-sudo** [comment]
Yes center crop. These are the images that are NOT being transformed during the training phase: 
![image](https://user-images.githubusercontent.com/57682653/150636562-c8d78b18-3d71-4e31-a4d0-869e824415d9.png)

These are images that are being transformed during the training phase:
![image](https://user-images.githubusercontent.com/57682653/150636706-25cd9297-a80c-46f1-88a4-b40735b802fb.png)


I am changing dataset in each epoch for better training(in one epoch I am using a transformed batch in another not transformed. Here is my code: please check it out to see if my approach for changing each dataset in batch is correct or not :) :  https://www.codepile.net/pile/klwjbJEX).

**Comment by bigmb** [comment]
Ok. So you re using center crop in your transformation. 
What happens in center crop is that it chooses the center part only. You don't have to use it in your problem. 
I used it becuase my problem required it to concentrate only on the middle part of the image. Rest of the information was useless. 
Thats not the case in your images.
Did you visualize the images with center crop? Try to check those images. Let me know if that's that's case after visualization of those center crop images.

**Comment by muhammedcanpirincci-sudo** [comment]
Thats exactly how i thought. Thank you so much. I solved problem in my mind thanks to you :)
